src/physics/initial_conditions/initial_condition.cpp

```diff
@@ -104,6 +104,29 @@ ::value(const dealii::Point<dim,real> &/*point*/, const unsigned int /*istate*/)
     return value;
 }
 
+// ========================================================
+// 1D BURGERS VISCOUS -- Initial Condition
+// ========================================================
+template <int dim, typename real>
+InitialConditionFunction_BurgersViscous<dim,real>
+::InitialConditionFunction_BurgersViscous (const unsigned int nstate)
+        : InitialConditionFunction<dim,real>(nstate)
+{
+    // Nothing to do here yet
+}
```

Comment:
It might be cleaner in this case, but there was a discussion with Pranshul earlier in this PR about moving some of the physics constructors from the header to the cpp for consistency throughout the code. So maybe we should just leave them in the cpp to keep things consistent.

Reply:
It belongs in the .cpp. The only case where it should be in the header is when the = default keyword is used. The one here has an initializer list, so it does some non-default behavior.